pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "newron"
-version = "0.1.7"
+version = "0.1.8"
```

Comment:
@pbnewron could not find a way to maintain this version file in sync with version.py